### DISCUSSION TITLE
Locating the Project Directory Pulled from GitHub

### DISCUSSION THREAD

**Discussion by khenpheakdey** [coollabsio/coolify#4580] [Q&A]
Does Coolify store the project directory pulled from the GitHub source somewhere?

I need to run some manual commands within my application, but I’m struggling to find the project root directory. Is there a specific location or method to determine where Coolify stores the cloned project files?

### Context
- I have already set up a GitHub repository integration in Coolify.
- The application is deployed and runs fine, but I can't figure out where the project directory resides.
- I have searched for related discussions but haven't found anything conclusive.

Any help would be appreciated!

**Comment by Julhol-droid** [comment]
@khenpheakdey  I am wondering this too. Have you found a solution?

**Reply by khenpheakdey** [reply]
Nope, I decided to use docker command instead.

**Comment by vjccruz** [comment]
If you need a solution, try:

> Projects > ProjectX > Configuration > General > Base Directory

This solution worked for me.

**Reply by khenpheakdey** [reply]
Great encounter, but what I'm looking for is the root directory of the project inside hosted server, which can only be found inside the container its self currently.

**Comment by Cinzya** [comment]
Coolify deletes your repo after finishing the build process.
For applications using the Docker Compose build pack, you have the option to disable that behaviour by enabling `Preserve Repository during Deployment`, then your files can be found under `/data/coolify/applications/<uuid>`.

<img width="752" height="312" alt="grafik" src="https://github.com/user-attachments/assets/672be949-73a1-47d4-b8c1-80c8cdd00aa2" />
